package.json

```diff
@@ -49,17 +53,19 @@
     "zustand": "^4.4.7"
   },
   "devDependencies": {
+    "@electron/rebuild": "^3.6.0",
     "@testing-library/react": "^14.1.2",
     "@types/react": "^18.2.42",
     "@types/react-dom": "^18.2.17",
     "@vitejs/plugin-react": "^4.2.1",
-    "electron": "^29.0.0",
+    "electron": "28.2.4",
```

Comment:
This is becasue of: https://github.com/WiseLibs/better-sqlite3/pull/1151
When merged, we can restore electron 29